src/ReactiveUI.Fody.Tests/ReactiveUI.Fody.Tests.csproj

```diff
@@ -1,26 +1,33 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
   <PropertyGroup>    
-    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);net461</TargetFrameworks>
+    <FodyTargetFramework>netstandard2.0</FodyTargetFramework>
+    <FodyTargetFramework Condition=" $(TargetFramework.StartsWith('net4')) ">$(TargetFramework)</FodyTargetFramework>
```

Comment:
Should this be `$(FodyTargetFramework)` ?

Reply:
No. In this case I want net461 (or whatever future versions we use)